.env.example

```
@@ -16,4 +16,8 @@ HEALTHCHECK_PORT=8080
 
 # Espo CRM
 ESPO_API_KEY=your_key_here
-ESPO_BASE_URL=https://espo.url/
\ No newline at end of file
+ESPO_BASE_URL=https://espo.url/
+
+# Kimai Time Tracking
+KIMAI_BASE_URL=https://kimai.example.com
+KIMAI_API_TOKEN=your_kimai_api_token_here
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)